ITEM.tsv

```diff
@@ -398,206 +398,206 @@ ITEM_20150317_000397	$Viper
+ITEM_20150317_000401	$Chapparition Bow 
+ITEM_20150317_000402	$The engravings say: 'Straighten your back and broaden your shoulders; pull the bowstring and push the bow as you shoot.' 
+ITEM_20150317_000403	$Blood Gain 
+ITEM_20150317_000404	$This bow was created by Lydia Schaffen. She frequently used this bow throughout her lifetime but intentionally witheld the crafting method from 'The Tower of the Stars' and 'Schaffenstar', two entities founded by Lydia herself. 
+ITEM_20150317_000405	$Tempest Shooter 
+ITEM_20150317_000406	$A new bow designed and developed through a joint effor of master bowcrafters. It was conceived with the hope of overcoming the Great Plant Cataclysm. 
+ITEM_20150317_000407	$Empty_weapon_Bow 
+ITEM_20150317_000408	$Low Light Crossbow 
+ITEM_20150317_000409	$Crossbow [Stab] 
+ITEM_20150317_000410	$Useful when having a free hand is more important than having more attack power. 
+ITEM_20150317_000411	$Low Crossbow 
+ITEM_20150317_000412	$Modern crossbows use many different materials, mostly monster parts. 
+ITEM_20150317_000413	$Dokyu 
+ITEM_20150317_000414	$A weapon that is easy to reload. 
+ITEM_20150317_000415	$Quarrel Bow 
+ITEM_20150317_000416	$The Quarrel Shooter Master uses this to teach beginners. 
+ITEM_20150317_000417	$Oak Crossbow 
+ITEM_20150317_000418	$Although craftable wood is hard to find since the Great Plant Cataclysm, this weapon is not hard to make. 
+ITEM_20150317_000419	$Seal Crossbow 
+ITEM_20150317_000420	$Because monster bones are sometimes cursed, a seal is attached to this weapon. 
+ITEM_20150317_000421	$Catapult 
+ITEM_20150317_000422	$There is a long history of research on adapting siege weaponry for single-person use. This weapon is the culmination of that research. 
+ITEM_20150317_000423	$Cranequine 
+ITEM_20150317_000424	$The mechanism assists beginners in using it, but experienced users will benefit from the added power. 
+ITEM_20150317_000425	$Crude Oak Crossbow 
+ITEM_20150317_000426	$Crude Seal Crossbow 
+ITEM_20150317_000427	$Sniper Crossbow 
+ITEM_20150317_000428	$A powerful crossbow, infamous for being used by the Assassin Evoniphon. 
+ITEM_20150317_000429	$Light Crossbow 
+ITEM_20150317_000430	$Heavy Light Crossbow 
+ITEM_20150317_000431	$Crossbow 
+ITEM_20150317_000432	$Heavy Dokyu 
+ITEM_20150317_000433	$Heavy Quarrel Bow 
+ITEM_20150317_000434	$Heavy Oak Crossbow 
+ITEM_20150317_000435	$Dunkel Quarrel Bow 
+ITEM_20150317_000436	$Dunkel Cranequine 
+ITEM_20150317_000437	$Dunkel Crossbow 
+ITEM_20150317_000438	$Dunkel Dokyu 
+ITEM_20150317_000439	$Dunkel Oak Crossbow 
+ITEM_20150317_000440	$Arbalest 
+ITEM_20150317_000441	$Fedimian Turret 
+ITEM_20150317_000442	$Bullet Shooter 
+ITEM_20150317_000443	$Black Tip Bow 
+ITEM_20150317_000444	$Spiked Crossbow 
+ITEM_20150317_000445	$Windless 
+ITEM_20150317_000446	$Heavy Crossbow 
+ITEM_20150317_000447	$Though slightly heavy, the magics infused into the bow itself offset this downside. 
```

Comment:
Hmm, I don't understand this change.  The original sentence is correctly structured, and properly illustrative through flavor text.